### PR TITLE
Add io.github.alainm23.planify

### PR DESCRIPTION
**Pull request by alainm23**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. https://github.com/alainm23/planner
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. 

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission
### I only want to use the beta branch for now, until we have a first stable release sometime in the future. We think that's a good way to keep users up to date to gather more user feedback.

image

### CODE REVIEW

**Comment by alainm23** [comment]
bot, build com.github.alainm23.planify

**Comment by alainm23** [comment]
bot, build com.github.alainm23.planify

**Comment by alainm23** [comment]
bot, build com.github.alainm23.planify

File: com.github.alainm23.planify.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
+{
+    "app-id" : "com.github.alainm23.planify",
```
Comment (hfiguiere) [inline com.github.alainm23.planify.json:2]: must be `io.github`

File: com.github.alainm23.planify.json [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    "tags" : [
+        "devel"
+    ],
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=session-bus",
```
Comment (hfiguiere) [inline com.github.alainm23.planify.json:16]: this is not allowed. you need to determine which "talk names" you need.

File: com.github.alainm23.planify.json [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        "--talk-name=org.gnome.evolution.dataserver.Calendar8",
+        "--talk-name=org.gnome.evolution.dataserver.Sources5",
+        "--metadata=X-DConf=migrate-path=/com/github/alainm23/planify/"
```
Comment (hfiguiere) [inline com.github.alainm23.planify.json:19]:
```suggestion
```

this is uncessary.

File: com.github.alainm23.planify.json [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dtracing=true",
+                "-Dprofile=development"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/alainm23/planner.git"
```
Comment (hfiguiere) [inline com.github.alainm23.planify.json:131]: you need a tag or a commit

File: com.github.alainm23.planify.json [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+                }
+            ]
+        },
+        {
+            "name" : "planify",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dtracing=true",
+                "-Dprofile=development"
```
Comment (hfiguiere) [inline com.github.alainm23.planify.json:126]: given that flathub is for users, I believe these build options might be incorrect.

File: com.github.alainm23.planify.json [hunk 6]
```diff
@@ -0,0 +1,136 @@
+{
+    "app-id" : "com.github.alainm23.planify",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.alainm23.planify",
+    "tags" : [
+        "devel"
+    ],
```
Comment (hfiguiere) [inline com.github.alainm23.planify.json:9]:
```suggestion
```

this is not needed

File: com.github.alainm23.planify.json (hunk 1 above)
Comment (hfiguiere) [reply com.github.alainm23.planify.json:2]: this is a fatal error

File: com.github.alainm23.planify.json (hunk 2 above)
Comment (hfiguiere) [reply com.github.alainm23.planify.json:16]: this is a fatal error

File: com.github.alainm23.planify.json (hunk 4 above)
Comment (hfiguiere) [reply com.github.alainm23.planify.json:131]: this is a fatal error

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by alainm23** [comment]
bot, build com.github.alainm23.planify

**Comment by alainm23** [comment]
bot, build io.github.alainm23.planify

**Comment by alainm23** [comment]
bot, build io.github.alainm23.planify

**Comment by alainm23** [comment]
bot, build io.github.alainm23.planify

**Comment by alainm23** [comment]
bot, build io.github.alainm23.planify

File: io.github.alainm23.planify.json [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+    "command" : "io.github.alainm23.planify",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gnome.evolution.dataserver.Calendar8",
+        "--talk-name=org.gnome.evolution.dataserver.Sources5",
+        "--talk-name=io.github.alainm23.planify"
```
Comment (hfiguiere) [inline io.github.alainm23.planify.json:15]: this name is granted by default
```suggestion
```

Comment (hfiguiere) [inline io.github.alainm23.planify.json:15]: this name is granted by default

```suggestion
        "--talk-name=org.gnome.evolution.dataserver.Sources5"
```

**Comment by alainm23** [comment]
bot, build io.github.alainm23.planify

**Pull request closed without merging**